sippy-ng/src/components/Sidebar.js

```diff
@@ -97,38 +97,6 @@ export default function Sidebar(props) {
       <CapabilitiesContext.Consumer>
         {(value) => {
           if (value.includes('build_clusters')) {
```

Comment:
This doesn't look right to me, it ends up putting the entire tools menu under the build_clusters capability.   You can move this whole block under Tools, starting with the CapabilitiesContext.Consumer, but instead of returning the Divider and List headers, just return the ListItem

Reply:
Yeah, what was unclear to me was if we wanted that nested inside of 
`  if (value.includes('openshift_releases')) {`

But let me play around with it more